src/network/webtask.cpp

```diff
@@ -236,6 +236,14 @@ void WebTask::slotStart(int timeoutMillis) {
             Qt::UniqueConnection);
 }
 
+void WebTask::slotPending() {
```

Comment:
Do you intend to connect these function to a signal? 
If not you can make them normal functions.

Reply:
Thank you for noticing me that. I'll do that.